src/clr-angular/modal/modal.spec.ts

```diff
@@ -206,6 +223,21 @@ describe('Modal', () => {
     })
   );
 
+  it(
+    'should not be closed on backdrop click by default',
+    fakeAsync(() => {
+      const defaultsFixture = TestBed.createComponent(TestDefaultsComponent);
+      defaultsFixture.detectChanges();
+      compiled = defaultsFixture.nativeElement;
+
+      const backdrop: HTMLElement = compiled.querySelector('.modal-backdrop');
+
+      backdrop.click();
+      flushAndExpectOpen(defaultsFixture, true);
+      defaultsFixture.destroy();
```

Comment:
Could this go run in after every unit test? Or it should run in specific tests?

Reply:
It is only needed here, because we're using a different test component, where the template is as stripped of inputs, as possible, so we can test defaults. That's why we have the `defaultsFixture` only here. All the other tests are using the normal `fixture` with the extended template, which has all needed inputs.